website_form_builder/static/src/css/website_form_builder.less

```
@@ -15,10 +15,12 @@
     }
 }
 
-body .modal.o_website_modal {
+/* Make modals with multiselect with a usable height */
```

```suggestion
// Make modals with multiselect with a usable height
```

In Scss, `/* these comments */` appear in the compiled CSS. That's why they are better suited for i.e. license comments (or just block comments as usual).

Since you are commenting over the code itself and it's a single line, it makes more sense to use `// Single line comment`.

It's not like this is a critical change, but I think you probably didn't know this little detail, so it's good to learn. :blush: